packages/simcore-sdk/src/simcore_sdk/node_data/data_manager.py

```diff
@@ -70,8 +71,9 @@ async def push(
             io_log_redirect_cb=io_log_redirect_cb,
         )
     # we have a folder, so we create a compressed file
-    with TemporaryDirectory() as tmp_dir_name:
-        log.info("compressing %s into %s...", file_or_folder.name, tmp_dir_name)
+    with log_catch(log), log_context(
+        log, logging.INFO, "pushing %s", file_or_folder

```

Comment:
only note here, the message changed

Reply:
which is correct, since it is what it is doing